src/AspNetCore/Src/applicationinfo.cpp

```diff
@@ -112,15 +112,10 @@ APPLICATION_INFO::UpdateAppOfflineFileHandle()
         // if it was, log that app_offline has been dropped.
         if (m_fAppOfflineFound)
         {
-            STACK_STRU(strEventMsg, 256);
-            if (SUCCEEDED(strEventMsg.SafeSnwprintf(
-                ASPNETCORE_EVENT_RECYCLE_APPOFFLINE_REMOVED_MSG)))
-            {
-                UTILITY::LogEvent(g_hEventLog,
-                    EVENTLOG_INFORMATION_TYPE,
-                    ASPNETCORE_EVENT_RECYCLE_APPOFFLINE_REMOVED,
-                    strEventMsg.QueryStr());
-            }
+            UTILITY::LogEventF(g_hEventLog,
+                EVENTLOG_INFORMATION_TYPE,
+                ASPNETCORE_EVENT_RECYCLE_APPOFFLINE_REMOVED,
+                ASPNETCORE_EVENT_RECYCLE_APPOFFLINE_REMOVED_MSG);
```

Comment:
for this one, please try to add the application info to the log. if there are lots applications, it will be hard to tell which one's app_offlinefile was removed based on the log.

Reply:
I'm not changing messages themselves in this PR. Just refactoring the LogEvent usages. We can file it and do it the future. I'd like to keep this mechanical.

Reply:
In such case, we should call LogEvent instead of LogEventF to avoid creating stru and string copy

Reply:
I'll change LogEventF to short circuit on empty args.

Reply:
Turns out it's hard, changed to LogEvent